website/content/api-docs/variables/variables.mdx

```diff
@@ -292,6 +296,7 @@ the response will include only metadata and not the `Items` field:
 
 
 [Variables]: /nomad/docs/concepts/variables
+[locks section]: /nomad/docs/commands/var/lock
```

Comment:
seems strange to link over to the command from the API doc.  maybe this is what you were going for instead?

```suggestion
[locks section]: /nomad/api-docs/variables/locks
```